test/index.js

```diff
@@ -134,18 +126,27 @@ describe('interpret.extensions', function() {
         }
       }
 
+      // Skip any swc test on linux due to https://github.com/swc-project/swc/issues/4107
```

Comment:
Skipping on linux until https://github.com/swc-project/swc/issues/4107 is resolved.